code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -73,6 +73,88 @@ GLOBAL_LIST_EMPTY(antagonists)
 	owner = null
 	return ..()
 
+/datum/antagonist/Topic(href,href_list)
+	if(!check_rights(R_ADMIN))
+		return
+
+	//Some commands might delete/modify this datum clearing or changing owner
+	var/datum/mind/persistent_owner = owner
+
+	var/commands = get_admin_commands()
+	for(var/admin_command in commands)
+		if(href_list["command"] == admin_command)
+			var/datum/callback/C = commands[admin_command]
+			C.Invoke(usr)
```

Comment:
you seem pretty keen on getting rid of single-letter variables in this PR, i think these ones must have slipped you by

Reply:
there's a few more scattered but i think that's just reorganization so whatever

Reply:
Definitely. Just wanted to keep the diff down since it's getting big. Cult constructs I moved but did not change, but I'm willing to change both tbh

Reply:
addressed all of these. constructs getting names and comments. some mild refactoring.